train.py

```diff
@@ -252,8 +256,12 @@ def main():
 
             if step % 50 == 0:
                 save(saver, sess, logdir, step)
+                last_save_step = step
 
     finally:
+        if step > last_save_step:
+            print()
```

Comment:
When you hit ctrl-C to kill it, a ^C is displayed. The print() puts the saving message on the next line after the ^C. Easier to read.